LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2012-2021 Scott Chacon and others
```

Comment:
Copyright @enarjord?

Reply:
Sorry! I removed it. It was a pasted copy of another project.